thumbfast.lua

```diff
@@ -502,3 +512,4 @@ mp.register_script_message("thumb", thumb)
 mp.register_script_message("clear", clear)
 
 mp.register_event("file-loaded", file_load)
+mp. register_event("shutdown", shutdown)
```

Comment:
Extra space.

Reply:
Didn't know there could be a space in between and it still works. Interesting